.github/workflows/cron.yml

```diff
@@ -1,63 +1,59 @@
 name: "Hourly Build & Deploy to Netlify"
 on:
   schedule:
-  - cron: '0 * * * *'
+    - cron: "0 0 * * *"
 jobs:
   hourly-build-and-deploy:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     steps:
-    - name: Checking out the repository
-      uses: actions/checkout@v3
-      with:
-        fetch-depth: 0
+      - name: Checking out the repository
+        uses: actions/checkout@v3
 
-    - name: Installing Nix
-      uses: cachix/install-nix-action@v22
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-        extra_nix_config: |
-          experimental-features = nix-command flakes
```

Comment:
Why do we disable `flakes`? Doesn't this break the... everything?

Reply:
It is enabled by default.